ExposureNotification.App/ExposureNotification.App/Services/ExposureNotificationHandler.cs

```diff
@@ -39,8 +40,10 @@ public async Task ExposureDetected(ExposureDetectionSummary summary, Func<Task<I
 			// Pop up a local notification
 		}
 
-		public async Task FetchExposureKeysFromServer(Func<IEnumerable<TemporaryExposureKey>, Task> addKeys)
+		public async Task<IEnumerable<TemporaryExposureKey>> FetchExposureKeysFromServerAsync()
```

Comment:
I am returning the full list here because iOS needs to make its own batched files... and soon Android. We could have the "add delegate" but we can't really control how many items the user sends... We could have a property and throw?

Reply:
I still think we should leave this as a delegate... We want the developer to be able to send these in batches as they download them in batches.

Reply:
We can still batch them further internally if they send us too many.

Reply:
Yep.